bcl2fastq/index_summary.py

```diff
@@ -5,66 +5,110 @@
 import dash.dependencies as dep
 import pandas
 
-
 index = gsiqcetl.bcl2fastq.parse.load_cache(
     gsiqcetl.bcl2fastq.parse.CACHENAME.SAMPLES,
     './data/bcl2fastq_cache.hd5'
 )
-
 unknown = gsiqcetl.bcl2fastq.parse.load_cache(
     gsiqcetl.bcl2fastq.parse.CACHENAME.UNKNOWN,
     './data/bcl2fastq_cache.hd5'
 )
 
 all_runs = index['Run'].sort_values(ascending=False).unique()
+
 all_runs = [{'label': r, 'value': r} for r in all_runs]
 
+# Within the main layout division there are 4 groups:
+#           1. the section with the 100% bar graph
+#           2.  Nested subdivision of:
+#                   a) pie graph at 24%
+#                   b) bar graph at 76%
+#           3. Sample Run hidden
+#           4. Pruned  Unknown Hidden
+# Note: span can be used for inline setting
+
+
+#           Dropdown option's label set to all_runs(combination of unknown and known)
+#           Dropdown multi set to false (default) - only single select possible
+#                 -  multiple select gives error due to inconsistent values
 layout = html.Div(children=[
     dcc.Dropdown(
         id='run_select',
         options=all_runs,
         value=all_runs[0]['value'],
+        clearable=False
     ),
+
     dcc.Graph(
         id='known_index_bar',
+
     ),
+    # Bar graph with defaults set, no change in colour etc
     html.Div([
         html.Div(
             [dcc.Graph(id='known_unknown_pie'),
              dcc.Textarea(
                  id='known_fraction',
                  style={'width': '100%'},
                  readOnly=True,
+                 # This is the textbox at the bottom, hover over to see title
                  title=(
                      "Assumptions are made about which indexes are known "
                      "or unknown. This is due to multiple bcl2fastq analyses "
                      "being used on one run. This number should be 100%."
                  ),
+
              )],
-            style={'width': '24%', 'display': 'inline-block'}
+            style={'width': '24%', 'display': 'inline-block', }
         ),
         html.Div(
             [dcc.Graph(id='unknown_index_bar')],
-            style={'width': '74%', 'display': 'inline-block', 'float': 'right'}
+            style={'width': '76%', 'display': 'inline-block', 'float': 'center'}
```

Comment:
I was trying to see if it would change the overlapping that occurs if you expand the text box in the bottom left, but forgot to change it back,I can do that in the next commit